[2024-04-18T00:16:46.224+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Units_statistics_DAG.ETL_job manual__2024-04-15T21:09:06.487421+00:00 map_index=13 [queued]>
[2024-04-18T00:16:46.434+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Units_statistics_DAG.ETL_job manual__2024-04-15T21:09:06.487421+00:00 map_index=13 [queued]>
[2024-04-18T00:16:46.434+0000] {taskinstance.py:2193} INFO - Starting attempt 12 of 12
[2024-04-18T00:16:46.543+0000] {taskinstance.py:2217} INFO - Executing <Mapped(_PythonDecoratedOperator): ETL_job> on 2024-04-15 21:09:06.487421+00:00
[2024-04-18T00:16:46.578+0000] {standard_task_runner.py:60} INFO - Started process 368833 to run task
[2024-04-18T00:16:46.648+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'Units_statistics_DAG', 'ETL_job', 'manual__2024-04-15T21:09:06.487421+00:00', '--job-id', '7006', '--raw', '--subdir', 'DAGS_FOLDER/Units_dag.py', '--cfg-path', '/tmp/tmpiiu4s6n7', '--map-index', '13']
[2024-04-18T00:16:46.675+0000] {standard_task_runner.py:88} INFO - Job 7006: Subtask ETL_job
[2024-04-18T00:16:47.055+0000] {task_command.py:423} INFO - Running <TaskInstance: Units_statistics_DAG.ETL_job manual__2024-04-15T21:09:06.487421+00:00 map_index=13 [running]> on host a09da322e808
[2024-04-18T00:16:47.403+0000] {taskinstance.py:2731} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 2360, in _run_raw_task
    self._execute_task_with_callbacks(context, test_mode, session=session)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 2498, in _execute_task_with_callbacks
    task_orig = self.render_templates(context=context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 2910, in render_templates
    original_task.render_template_fields(context)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/mappedoperator.py", line 799, in render_template_fields
    mapped_kwargs, seen_oids = self._expand_mapped_kwargs(context, session)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/decorators/base.py", line 548, in _expand_mapped_kwargs
    op_kwargs, resolved_oids = super()._expand_mapped_kwargs(context, session)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/mappedoperator.py", line 658, in _expand_mapped_kwargs
    return self._get_specified_expand_input().resolve(context, session)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/expandinput.py", line 202, in resolve
    data = {k: self._expand_mapped_field(k, v, context, session=session) for k, v in self.value.items()}
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/expandinput.py", line 202, in <dictcomp>
    data = {k: self._expand_mapped_field(k, v, context, session=session) for k, v in self.value.items()}
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/expandinput.py", line 188, in _expand_mapped_field
    raise TypeError(f"can't map over value of type {type(value)}")
TypeError: can't map over value of type <class 'NoneType'>
[2024-04-18T00:16:47.446+0000] {taskinstance.py:2777} ERROR - Unable to unmap task to determine if we need to send an alert email
[2024-04-18T00:16:47.446+0000] {taskinstance.py:1149} INFO - Marking task as FAILED. dag_id=Units_statistics_DAG, task_id=ETL_job, map_index=13, execution_date=20240415T210906, start_date=20240418T001646, end_date=20240418T001647
[2024-04-18T00:16:47.502+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 7006 for task ETL_job (can't map over value of type <class 'NoneType'>; 368833)
[2024-04-18T00:16:47.549+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-04-18T00:16:47.604+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
